packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -289,6 +290,14 @@ export const HelpCenterContactForm = () => {
 						`Plan: ${ productId } - ${ productName } (${ productTerm })`,
 					];
 
+					if ( getQueryArgs()?.ref === 'woocommerce-com' ) {
```

Comment:
I really dislike query params as they're impossible to type and document and statistically analyze, the biggest source of bugs. But I thought about this, and I can't come up with a better way.

Reply:
I agree with your point. I think at the very least, we could leave a comment on this [line](https://github.com/Automattic/woocommerce.com/blob/fb06c9d179895b0d865a5cde00c2c0cbc4702eb8/themes/woo/start/components/installation/installation.js#L71) to make a reference to this PR with a note about changing the parameter.

Reply:
> we could leave a comment on this [line](https://github.com/Automattic/woocommerce.com/blob/fb06c9d179895b0d865a5cde00c2c0cbc4702eb8/themes/woo/start/components/installation/installation.js#L71) to make a reference to this PR with a note about changing the parameter.

That's a good idea!

Reply:
Issue created https://github.com/Automattic/woocommerce.com/issues/15607